### DISCUSSION TITLE
termostati e valvole Zigbee

### DISCUSSION THREAD

**Discussion by PiuCurioso** [indomus/forum#587] [ZigBee, Z-Wave, Thread, Matter]
Carissimi buongiorno vi volevo chiedere un consiglio: stavo pensando di acquistare un cronotermostato e delle valvole termostatiche (da gestire in locale, non devono accedere ad internet e quindi ai server dei produttori)per domotizzare l'impianto di riscaldamento, in quanto devo acquistare una nuova caldaia a condensazione.  Mi interesserebbero dei dispositivi Zigbee compatibili con il software Gateway ZigBee2MQTT +mosquitto.Grazie per i vostri consigli Un caro saluto

**Comment by snakuzzo** [comment]
Alcuni utenti su Telegram ci hanno detto che stanno usando le "TUYA Zigbee TRV termostato valvola termostatic"...o almeno così puoi cercarle su Aliexpress. Di seguito uno screenshot (non sono ammessi link esterni)

![image](https://user-images.githubusercontent.com/7205371/128186891-d1c7f422-b2b5-412f-a45d-775ac10971a7.png)

**Comment by PiuCurioso** [comment]
Ok Snakuzzo grazie assai per l'imput ma di questa marca esiste anche il cronotermostato?in quanto su ogni termosifone metterei la valvola ma poi dovrei installare anche un cronotermostato da cui poter comandare la caldaia ,accenderla spegnerla programmarla etc etc.grazie mille buon pranzo
Ciao ciao

**Reply by maurominella** [reply]
Effettivamente il tema non è banale, ho aperto una discussione ad hoc su questo forum. Non posso mettere link ma la trovate sostituendo nell'URL della pagina il numero 624. Buona continuazione

**Reply by snakuzzo** [reply]
Perché dici che non puoi mettere il link al tuo topic ?

**Comment by maurominella** [comment]
Ah si può? In genere nei forum bloccano gli URL. Allora eccolo qui: https://github.com/indomus/forum/discussions/624. Grazie!

**Reply by M0ntyC0r4** [reply]
avevo dato una letta a questa discussione ed uno dei problemi che elenchi lì dovresti poterlo risolvere forzando la temperatura rilevata dalla valvola.  Sulle mie, con una combinazione di tasti (tasto centrale + tasto -) accedi ad un menu avanzato ed una delle opzioni riguarda la temperatura rilevata dalla valvola, che puoi forzare in base ad un range da +9 a -9 (almeno, sulle mie).

Se metti un termometro al centro della stanza ed accendi i riscaldamenti, puoi regolare la temperatura rilevata dalle valvole e vedi lo scarto rispetto a quella rilevata al centro della stanza, così regoli la valvola: metti che il termometro al centro della stanza rileva 21 e la valvola 26, tu gli imposti "-5" ed hai risolto. Poi vai in un'altra stanza e fai la stessa cosa.

Guarda sulle istruzioni delle tue valvole se ci sta questo menu avanzato di cui parlo io!! XD

**Reply by snakuzzo** [reply]
Quello si chiama "offset" e comunque non si risolve così facilmente la differenza di temperatura rilevata...non è una cosa così banale.
Dico questo perché non è detto che lo scarto sia sempre quello rilevato la prima volta. La cosa giusta sarebbe allineare l'offset a cadenza regolare recuperando la temperatura da un sensore posizionato in altro punto (quello che ti pare)...magari tramite HA

**Reply by M0ntyC0r4** [reply]
> Quello si chiama "offset" e comunque non si risolve così facilmente la differenza di temperatura rilevata...non è una cosa così banale. Dico questo perché non è detto che lo scarto sia sempre quello rilevato la prima volta. La cosa giusta sarebbe allineare l'offset a cadenza regolare recuperando la temperatura da un sensore posizionato in altro punto (quello che ti pare)...magari tramite HA

ecco sì...c'avevo pensato!! Se ogni volta lo scarto è diverso ti frega. Allora la cosa, giustamente, è spendere soldi per un sensore temperatura in ogni stanza e legare le automazioni delle valvole a quello, invece che al sensore temperatura della valvola.

Oppure, per spendere meno, mettere un solo sensore  in corridoio (o comunque nel punto di casa che si scalda per ultimo) e legare le automazioni a quello dicendogli "quando QUELLO ha raggiunto tot gradi", e quindi in teoria è calda tutta casa, "chiudi le valvole".

**Reply by M0ntyC0r4** [reply]
Invece per accenderle e fare in modo che non si spengano (mi pare fosse un altro dei problemi), si potrebbe aggirare l'ostacolo accendendole e impostando una temperatura così alta che non la potrà mai raggiungere, forzandone in questo modo l'accensione.
Però, siccome da me ci sta il centralizzato, io non saprei dire se con la caldaia finiresti per consumare di più in questo modo...a rigor di logica, la possibilità ci sta tutta!!

Io non ho questo problema perché la temperatura massima non la gestisco io ma direttamente il fornitore del servizio...quindi a me l'acqua nei termosifoni arriva già a tot gradi e io non posso né aumentarli né diminuirli!!

**Reply by snakuzzo** [reply]
No...non va bene così.
Per fare come dici dovresti avere valvole che oltre ad OFF prevedano uno stato ON senza chiudere la valvola a temperatura target raggiunta.

**Reply by M0ntyC0r4** [reply]
> No...non va bene così. Per fare come dici dovresti avere valvole che oltre ad OFF prevedano uno stato ON senza chiudere la valvola a temperatura target raggiunta.

ma se la temperatura è tipo 35 gradi, impostando lo scarto direttamente sul trv, non li raggiungerà mai...quindi non andrà mai su off, finché non intervieni tu con un'automazione su ha per farla spegnere.

**Reply by M0ntyC0r4** [reply]
io ho fatto così per esempio...ho impostato su tutte le valvole uno scarto di -4 e la mia automazione ha prevede climate.turn_on e climate.set_temperature: 32°, che non raggiunge mai e quindi non mi si chiude, a meno che non intervenga l'altra automazione che ho fatto, per spegnere i termosifoni!

**Reply by snakuzzo** [reply]
Questo lo faccio anche io con le valvole Tadoº dinamicamente, visto che la temperatura target è massimo 25⁰C.
Mi serve in inverno per eventuale biancheria da asciugare in breve tempo :D, che in estate per lasciare le valvole sempre aperte (visto che tenendole chiuse per troppo tempo, dopo mesi di inutilizzo potrebbero rimanere incastrate...cosa che già mi è capitata).
Resta il fatto che sarebbe più "pulito", ove possibile, gestire l'offset dinamicamente e a cadenza regolare basandosi su un sensore esterno.
In quel caso avresti bisogno della sola automazione per la sincronizzazione delle temperature, tutto il resto, inclusa la gestione - - manuale delle valvole, sarebbe gestita in maniera semplice...

**Reply by M0ntyC0r4** [reply]
Si lo penso anche io...però ti scontri con la compatibilità dei dispositivi che non sempre è al 100%.

Oddio, io in realtà potrei installare tipo l'addon tuya smart life e vedere se così facendo aumenta il grado di compatibilità (sicuramente, non ho dubbi) ma non è utile, visto che basta avviarle e subito deconz le legge come entità climate.

Dà errore quando gli dai il comando set_temperature ma lo esegue, tanto che sulla valvola imposta la temperatura corretta, quindi non me ne sono preoccupato più di tanto.

Comunque, tante volte devi fare i conti con una compatibilità non proprio pienissima e quindi girare intorno ai problemi, usando un po' di creatività.

Infatti alla fine anche a lui, con lo script che ha fatto, mi sembra funzioni tutto!!

**Comment by vpomax** [comment]
Qualcuno sta utilizzando queste termovalvole con Home Assistant??
![rv05b](https://user-images.githubusercontent.com/33105151/136939184-48bad348-7d97-4f09-84c8-657f92f545d1.png)

**Comment by M0ntyC0r4** [comment]
Io conosco questo sito che vi dice tutti i dispositivi compatibili con home assistant e con quale gateway zigbee: https://zigbee.blakadder.com/index.html

Personalmente ho di recente comprato queste: https://zigbee.blakadder.com/Beca_BRT-100.html

Mi trovo, per ora, abbastanza bene; parlano con conbee II (deconz) "nativamente", anche se con funzionalità limitate ad acceso, spento (imposta la temperatura a 5 gradi e quello, per lui, è spento ma il termosifone è gelido, quindi funziona) e auto (che presuppone tu abbia impostato le fasce orarie su ogni singola valvola, cosa che non fai perché puoi farlo con un automazione di ha).

Io ho riscaldamento centralizzato nel condominio ma con termocontabilizzatori in ogni appartamento, per la ripartizione dei consumi, quindi posso già impostare le delle fasce orarie in cui dire all'impianto di accendersi o spegnersi; questi TRV, perciò, mi servono per controllare ogni singolo elemento in modo separato, cosa che altrimenti non potrei fare, se non manualmente.

Ho impostato un'automazione che chiude il termosifone di una stanza, se la finestra è aperta ed un' automazione che, se siamo tutti fuori casa, forza lo spegnimento di tutti i termosifoni dell'appartamento (che altrimenti, se in fascia oraria, partirebbero a pieno regime).

**Reply by KC-inDomus** [reply]
il problema di questa roba è che che costa 20 euro e che vale, 20 euro. imho.

**Reply by M0ntyC0r4** [reply]
> il problema di questa roba è che che costa 20 euro e che vale, 20 euro. imho.

oddio, non sarà di qualità eccelsa...detto questo sul sito ufficiale della moes, su amazon, ebay e qualsiasi altro portale non cinese le abbia cercate, stanno 60 euro a valvola, altro che 20...

**Reply by KC-inDomus** [reply]
sì per carità. dico solo che secondo me ne vale 20, non 60...

**Reply by M0ntyC0r4** [reply]
> sì per carità. dico solo che secondo me ne vale 20, non 60...

oddio, guarda (premesso che su aliexpress le paghi meno di 30) la pensavo anche io così ma devo dire che sono rimasto piacevolmente sorpreso dall'acquisto di queste valvole specifiche.
Fanno quello che devono e, soprattutto, sono facilissime da accoppiare con deconz...praticamente plug&play.
Forse la tenuta delle batterie non mi ha convinto in pieno...comunque è ancora molto presto, perché le ho da una settimana...se nel tempo dovessi cambiare idea cercherò di ricordarmi di dare un feedback ma per ora sono abbastanza contento e le consiglierei a chi ha una configurazione simile alla mia!!!

**Reply by KC-inDomus** [reply]
mi fido assolutamente 👍

**Reply by M0ntyC0r4** [reply]
> mi fido assolutamente 👍

Da quello che leggo in giro, da monitorare sarà l'usura nel tempo, perché comunque sono sempre pezzi di plastica molto vicini ad una fonte di calore e perché alla fine il loro fare apri/chiudi di continuo sicuramente alla lunga le usurerà. Qui sul forum ho letto di un utente a cui se ne sono rotte 16 in 18 mesi...non so quali di preciso avesse comprato ma mi auguro che le mie siano un tantino più longeve...quello sì...

**Reply by KC-inDomus** [reply]
io di questo parlavo :) vabbè, incrocia le dita, si vedrà
